apis/virtualKubelet/v1alpha1/namespacemap_types.go

```diff
@@ -20,21 +20,25 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type mappingPhase string
+// MappingPhase indicates the status of the remote namespace.
+type MappingPhase string
 
 const (
 	// MappingAccepted indicates that a remote namespace is successfully created.
-	MappingAccepted mappingPhase = "Accepted"
+	MappingAccepted MappingPhase = "Accepted"
 	// MappingRefused indicates that at the moment is impossible to create a remote namespace.
-	MappingRefused mappingPhase = "Refused"
+	MappingRefused MappingPhase = "Refused"
```

Comment:
I would propose `CreationLoopBackOff` in case you are planning to implement a sort of backoff mechanism, otherwise `Refused` appears to mean it failed and no further attempt will be done.

Reply:
It sounds good, thank you very much.